chart/openfaas/README.md

```diff
@@ -283,6 +283,10 @@ Additional OpenFaaS options in `values.yaml`.
 | `operator.createCRD` | Create the CRD for OpenFaaS Function definition | `true` |
 | `ingress.enabled` | Create ingress resources | `false` |
 | `faasnetes.httpProbe` | Use a httpProbe instead of exec | `false` |
+| `ingressOperator.create` | Create the ingress-operator component | `true` |
```

Comment:
I thought we were going to go with `false` by default?

Reply:
Fixed.